File: unix/boot/spp/rpp/ratlibr/fcopy.r
include	defs

# fcopy - copy file in to file out

   subroutine fcopy (in, out)
   filedes in, out

   character line (MAXLINE)

   integer getlin

   while (getlin (line, in) != EOF)
      call putlin (line, out)

   return
   end

File: unix/boot/spp/rpp/ratlibf/fcopy.f
      subroutine fcopy (in, out)
      integer in, out
      integer line (128)
      integer getlin
23000 if (.not.(getlin (line, in) .ne. -1))goto 23001
      call putlin (line, out)
      goto 23000
23001 continue
      return
      end
